packages/node-build-scripts/generate-icons-source

```diff
@@ -101,22 +106,34 @@ function exportIconConsts(valueGetter) {
     return ICONS_METADATA.map(icon => `export const ${toEnumName(icon)} = "${valueGetter(icon)}";`);
 }
 
-/**
+/*
  * Loads SVG file for each icon, extracts path strings `d="path-string"`,
- * and constructs map of icon name to array of path strings.
- * @param {string} objectName
+ * and returns a map of icon names to stringified arrays of path strings.
  * @param {16 | 20} size
  */
-async function buildPathsObject(objectName, size) {
-    return Promise.all(
+async function buildPaths(size) {
+    const paths = {};
+    await Promise.all(
         ICONS_METADATA.map(async icon => {
             const filepath = path.resolve(__dirname, `../../resources/icons/${size}px/${icon.iconName}.svg`);
             const svg = fs.readFileSync(filepath, "utf-8");
             const pathStrings = await svgo
                 .optimize(svg, { path: filepath })
                 .then(({ data }) => data.match(/ d="[^"]+"/g) || [])
                 .then(paths => paths.map(s => s.slice(3)));
-            return `    "${icon.iconName}": [${pathStrings.join(",\n")}],`;
+            paths[icon.iconName] = `[${pathStrings.join(",\n")}]`;
```

Comment:
this is now a `reduce`, not a `map`.

`ICONS_METADATA.reduce(async (paths, icon) => { ... }, {})`